skiko/src/awtMain/kotlin/org/jetbrains/skiko/redrawer/MetalRedrawer.kt

```diff
@@ -154,13 +155,14 @@ internal class MetalRedrawer(
         windowOcclusionStateChannel.trySend(isOccluded)
     }
 
-    private fun performDraw() = synchronized(drawLock) {
+    private fun performDraw(waitVsync: Boolean = true) = synchronized(drawLock) {
```

Comment:
We don't call `performDraw = false` anywhere. I suppose we should change `redrawImmediately`

Reply:
Good catch! I missed it during refactoring